src/style/ngcolombia.material.scss

```diff
@@ -1,97 +1,120 @@
 @import '~@angular/material/theming';
 @import 'palette';
 @include mat-core();
+
 $primary: mat-palette($md-ngcolombiablue);
-$accent:  mat-palette($md-ngcolombia);
-$warn:  mat-palette($mat-red, 600, 100, 800);
+$accent: mat-palette($md-ngcolombia);
+$warn: mat-palette($mat-red, 600, 100, 800);
 $primary-color: mat-color($accent, 900);
 $temporaryprimary: mat-palette($md-ngcolombiabluetest);
 $ngcolombia-app-theme: mat-light-theme($temporaryprimary, $accent, $warn);
- 
-
 
 @include angular-material-theme($ngcolombia-app-theme);
 
-mat-icon, .material-icons {
-	font-family: 'Material Icons';
-	font-weight: normal;
-	font-style: normal;
-	font-size: 24px;  /* Preferred icon size */
-	display: inline-block;
-	line-height: 1;
-	text-transform: none;
-	letter-spacing: normal;
-	word-wrap: normal;
-	white-space: nowrap;
-	direction: ltr;
-  
-	/* Support for all WebKit browsers. */
-	-webkit-font-smoothing: antialiased;
-	/* Support for Safari and Chrome. */
-	text-rendering: optimizeLegibility;
-  
-	/* Support for Firefox. */
-	-moz-osx-font-smoothing: grayscale;
-  
-	/* Support for IE. */
-	font-feature-settings: 'liga';
-  }
+// Custom NgColombia Material Theme styles
+mat-icon,
+.material-icons {
+  font-family: 'Material Icons';
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px; /* Preferred icon size */
+  display: inline-block;
+  line-height: 1;
+  text-transform: none;
+  letter-spacing: normal;
+  word-wrap: normal;
+  white-space: nowrap;
+  direction: ltr;
 
-  /* Rules for sizing the icon. */
-mat-icon.md-18 { font-size: 18px; }
-mat-icon.md-24 { font-size: 24px; }
-mat-icon.md-36 { font-size: 36px; }
-mat-icon.md-48 { font-size: 48px; }
+  /* Support for all WebKit browsers. */
```

Comment:
Angular has `autoprefixer`, just make sure these vendor-support properties are indeed necessary.

Reply:
I didn't do those lines, that was auto format.

Reply:
Ok